### DISCUSSION TITLE
App crashing

### DISCUSSION THREAD

**Discussion by Shaashank009** [Shabinder/SpotiFlyer#162] [General]
Hello brother, App is crashing everytime this problem started yesterday not downloading anything.

**Comment by mondlicht-und-sterne** [comment]
for the programmer it would be helpful when exactly or approximately when the app crashes. so right at the start of the app or a little later?

do you know adb ? you can do this with log files, but it's very time-consuming, you could try it with scoop.

https://f-droid.org/packages/taco.scoop/

I haven't tested this tool, but it should help to show the programmer approximately where the error should be.

**Reply by thaynos** [reply]
i have this same issue as mentioned in 
#172 
whats adb?

**Reply by mondlicht-und-sterne** [reply]
> whats adb?

Android Debug Bridge (ADB)

**Reply by thaynos** [reply]
im using windows

**Comment by mondlicht-und-sterne** [comment]
It didn't say that you meant the windows version.

**Comment by thaynos** [comment]
?
